kytos/core/napps/base.py

```diff
@@ -241,6 +241,12 @@ def run(self):
 
         It should not be overriden.
         """
+        # Inform this NApp has been loaded
+        from kytos.core import KytosEvent
+        name = f'{self.username}/{self.name}.loaded'
+        event = KytosEvent(name=name, content={})
+        self.controller.buffers.app.put(event)
```

Comment:
1. The import is here because it would case a circular import problem?
1. May these 4 lines could be refactored in a separate method.

Reply:
Yes, the import is there to avoid a circular import.. I'll put that code on a separate method.